Client/Frontend/Library/LibraryViewController/LibraryViewController.swift

```diff
@@ -211,6 +212,9 @@ class LibraryViewController: UIViewController, Themeable {
                               accessibilityLabel: accessibilityLabel,
                               accessibilityIdentifier: accessibilityId)
             self.showPanel(navigationController)
+            if CoordinatorFlagManager.isLibraryCoordinatorEnabled {
+                navigationHandler?.start(panelType: viewModel.selectedPanel ?? .bookmarks, navigationController: navigationController)
+            }
```

Comment:
We can't put that yet in the `else`? If not, what are we missing to be able to?
```suggestion
            } else {
                setupLibraryPanel(panelVC, [...])
                self.showPanel(navigationController)
            }
```

Reply:
The show library panel embeds the child controller in the library view controller that's not done by the coordinator, which creates only the child coordinator.

Reply:
So we discussed we could probably embed the child VC from the coordinator instead. This won't be done in this PR for sure, so let's merge this for now.